templates/client.ts.hbs

```diff
@@ -12,6 +10,12 @@ import ResponseError from "./responseError";
 import TemplateURL from "./templateUrl";
 import { USER_AGENT_HEADER, USER_AGENT_VALUE } from "./version";
 
+if(isNode || !isBrowser || !hasFetchAvailable) {
```

Comment:
I'm not sure I entirely understand this conditional.

I think, if we're in a Node.js environment, we want to load `node-fetch`. If you're not in a Node environment, and you don't have `fetch`, well, the jig is up. We could consider throwing an error?

Reply:
I have edited the logic to throw an error if we are not in node but fetch is not defined.